sources/web/datalab/polymer/index.dark.css

```diff
@@ -12,16 +12,13 @@
  * the License.
  */
 
-body {
-  margin: 0px;
-  padding: 0px;
-  height: 100%;
-}
-datalab-app {
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  min-width: 470px;
+:root {
+  --primary-accent-color: #222;
+  --primary-bg-color: #222;
+  --secondary-bg-color: #2b2b2b;
+  --primary-fg-color: #aaa;
```

Comment:
The order seems a bit random here. Maybe you could put all the primary colors together, or put all the backgrounds together, or use alpha order.